### ISSUE TITLE
Filter posts based on sender's account

### ISSUE THREAD

**Issue by rupertoverall** [mastodon/mastodon#22603]
### Pitch

At the moment, it seems only possible to 'mute' accounts (which also hides any mentions of them) or 'filter' based on keywords (but not sender's account name). Neither of these options work for managing high-content feeds.
Ideally, it would be possible to hide a high-content account from the home timeline and only have it visible in a list. Mentions or boosts involving this account would still end up in the timeline.

This could be achieved if accounts could be silenced in the home timeline using modified filter criteria ("hide posts from where sender=$account").

### Motivation

I follow several bots from academic journals that post new content regularly. This content overwhelms my home timeline so that informative posts from individuals are often lost. I have a list containing all of these feeds which is great, but I can't hide this content from my timeline. I do not want to mute the feed account as that would also hide mentions of the posts (if colleagues boost or comment on this content, then tha is interesting to me). I can filter based on keyword, but the only common keyword is the account name (which also filters out boosts and mentions like muting does).

There are solid tools for user protection, which is great - but it would be helpful to have more support for content management as well.

**Comment by ClearlyClaire** [comment]
It seems like your first choice would be for #16556 to be fixed, which is kind of planned.

**Comment by rupertoverall** [comment]
> It seems like your first choice would be for #16556 to be fixed, which is kind of planned.

Yes - this sounds exactly what I am after. If this is really ready to be merged, then we can close my (apparently duplicate) request.

**Comment by ClearlyClaire** [comment]
Closing as a duplicate then, even though your suggestion was actually a bit different.

**Comment by lookitmychicken** [comment]
Can this be reopened? I maintain an account where I regularly post pictures. 

Someone has asked if there's some way to filter this kind of account out of their home feed by default, while still following the picture-posting account. 

I can suggest to the user that they set up a list of the accounts they most want to see, and use that as their default ... but if it was possible to add an account handle to a filter, then they could set up a filter that automatically adds a CW to any posts from the pic-posting account.

(Most followers of the pic-posting account don't want the pics to be behind a CW, but this particular follower would prefer it.)

**Comment by ClearlyClaire** [comment]
Reopening it, although there is some overlap with the “exclusive list” suggestion still.
